uaclient/data_types.py

```diff
@@ -11,6 +11,12 @@
 INCORRECT_FIELD_TYPE_ERROR_MESSAGE = (
     'Got value with incorrect type for field "{key}": {nested_msg}'
 )
+INCORRECT_ENUM_VALUE_TYPE_ERROR_MESSAGE = (
```

Comment:
I think we can remove this since it is already on #2147

Reply:
Oh yes, they slipped through after deleting everything else.